apps/web/src/pages/Applications/ApplicantDashboardPage/components/TrackApplications/TrackApplications.tsx

```diff
@@ -0,0 +1,286 @@
+        <p data-h2-margin="base(x.5, 0, 0, 0)">
+          {intl.formatMessage({
+            defaultMessage:
+              "Applications to talent pool and ongoing recruitment opportunities can be managed and tracked here. You’ll be able to see submission deadlines, your application’s status over time, and past applications.",
+            id: "iutl39",
+            description:
+              "Description for the track applications section on the applicant dashboard, paragraph one.",
+          })}
+        </p>
+        <p data-h2-margin="base(x.5, 0, x1, 0)">
+          {intl.formatMessage({
+            defaultMessage:
+              "After an application is successfully assessed, the talent pool will be added to your résumé automatically.",
+            id: "682ljn",
+            description:
+              "Description for the track applications section on the applicant dashboard, paragraph two.",
+          })}
+        </p>
+      </div>
+      <div>
+        <Accordion.Root
+          type="multiple"
+          mode="simple"
+          value={currentAccordionItems}
+          onValueChange={(newValue: AccordionItems) => {
+            setCurrentAccordionItems(newValue);
+          }}
+        >
+          {/* applications in progress */}
+          <Accordion.Item value="in_progress">
+            <StandardAccordionHeader headingAs="h3">
+              {currentAccordionItems.includes("in_progress")
+                ? intl.formatMessage(
+                    {
+                      defaultMessage:
+                        "Hide applications in progress ({applicationCount})",
+                      id: "1u9PTo",
+                      description:
+                        "Heading for applications in progress accordion on the applicant dashboard.",
+                    },
+                    {
+                      applicationCount: inProgressApplications.length ?? "0",
+                    },
+                  )
+                : intl.formatMessage(
+                    {
+                      defaultMessage:
+                        "Show applications in progress ({applicationCount})",
+                      id: "VUbrMi",
+                      description:
+                        "Heading for applications in progress accordion on the applicant dashboard.",
+                    },
+                    {
+                      applicationCount: inProgressApplications.length ?? "0",
+                    },
+                  )}
+            </StandardAccordionHeader>
+            <AnimatedContent
+              isOpen={currentAccordionItems.includes("in_progress")}
+            >
+              {intl.formatMessage({
+                defaultMessage:
+                  "This section contains your drafts and submitted applications that are still being processed.",
+                id: "WuMMRK",
+                description:
+                  "Introductory text displayed in applications in progress section.",
+              })}
```

Comment:
This should be passed to the `subtitle` prop on the `StandardAccordionHeader` so it appears even while the item is collapsed.

Reply:
:+1: [remove redundant attribute](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6823/commits/1f7a8d2c6ef34bdf20e4477ba4b6357b8fdcf43a)

Reply:
Good catch, I missed that it is still visible when closed.  It's blue instead of grey, but that's consistency.
Ref: [fix subtitles](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6823/commits/4fe65dc2a69398fa19ef71b5d978d95ad6fa88df)

Reply:
Insufficient contrast - Accordion subtitle #6898